plugins/index.js

```diff
@@ -119,7 +120,7 @@ export function removePlugin(manifest) {
     if (plugin && plugin.uninitialize) {
         plugin.uninitialize();
 
-    // Support the deprecated deinitialize callback from the plugins beta.
+        // Support the deprecated deinitialize callback from the plugins beta.
```

Comment:
Unrelated changes in this file?

Reply:
totally unrelated, I have no idea how that got in. Sorry, will fix.

Reply:
@jwilander better now